theme/helpers.js

```diff
@@ -9,13 +13,81 @@ exports.Color = class {
         Object.keys(shades).forEach((shade) => (this[shade] = shades[shade]));
     }
 
-    shade(shade) {
+    alter(shade = 500) {
 
-        return this[shade];
+        return Color(this[shade]);
     }
 
     toString() {
 
         return this['500'];
     }
 };
+
+exports.paletteFromTheme = (theme) => {
+
+    const colors = Object.keys(theme).reduce((acc, key) => {
+
+        const regex = /^color-(?<color>\w+)-(?<transparent>\w+-)?(?<shade>\d{3})/;
```

Comment:
I saw in UI Kitten's docs they have other naming conventions for colors e.g. `background-basic-color`. Why don't support any of those? Assuming I'm missing something, I didn't even know how MUI theming worked, let alone all this new business 🤡

Reply:
This is specifically built to support Eva's [color palette generator](https://colors.eva.design/?utm_campaign=eva_colors%20-%20home%20-%20eva_design%20website&utm_source=eva_design&utm_medium=referral&utm_content=eva_website_menu), which spits out colors that follow this convention.

Reply:
Ahhh ok gotcha. thanks for clarifying! 🙏